src/main/java/qna/domain/Answer.java

```diff
@@ -1,24 +1,38 @@
 package qna.domain;
 
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
-import javax.persistence.*;
-import java.util.Objects;
-
 @Entity
 public class Answer extends AuditingEntity {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+  
     @Lob
     private String contents;
 
     @Column(nullable = false)
     private Boolean deleted = false;
-    private Long questionId;
 
-    private Long writerId;
+    @ManyToOne
+    @JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```

Comment:
```suggestion
    @JoinColumn(foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```

name이 없다면 어떻게 동작하는지 알고 계실까요?

Reply:
몰랐습니다만 이것저것 적용해보니 새로운 사실을 알게 되었습니다.
별도로 지정하지 않으면
"[해당 엔티티의 컬럼명]_[연관관계에 있는 엔티티의 id 컬럼명]"
으로 작명되기 때문에
현재 상황에서는 name을 붙이지 않아도 동일한 fk이름이 적용됩니다.